imap_processing/cdf/config/shared/default_global_cdf_attrs_schema.yaml

```
@@ -0,0 +1,238 @@
+DOI:
+  description: >
+    DOI is a persistent Unique Digital Identifier with the form
+    https://doi.org/<PREFIX>/<SUFFIX> with the <PREFIX> identifying the DOI
+    registration authority and the <SUFFIX> identifying the dataset. The DOI should point to
+    a landing page for additional information about the dataset. DOIs are typically created by
+    the SPASE naming authority or archive.
+  default: null
+  required: false # NOT Required in ISTP Guide (Recommended)
+  validate: true
+  overwrite: false
+Data_level:
+  description: >
+    This attribute is used in file name creation and records the level of processsing done
+    on the dataset. For HERMES the following are valid values:
+    - l0>Level 0
+    - l1>Level 1
+    - l2>Level 2
+    - l3>Level 3
+    - l4>Level 4
+    - ql>Quicklook
+  default: null
+  required: true # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_product_descriptor:
+  description: >
+    This is an optional field that may not be needed for all products. Where it is used, identifier
+    should be short (e.q. 3-8 characters) descriptors that are helpful to end- users. If a
+    descriptor contains multiple components, underscores are used to separate those components.
+  default: null
+  required: false # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_type:
+  description: >
+    This attribute is used by CDF file writing software to create a filename. It is a
+    combination of the following filename components: mode, data level, and optional data
+    product descriptor.
+  default: null
+  required: false # NOT Required in ISTP Guide (Derived)
+  validate: false
+  overwrite: true
+Data_version:
+  description: >
+    This attribute identifies the version (vX.Y.Z) of a particular CDF data file.
+  default: null
+  required: true
+  validate: true
+  overwrite: false
+Descriptor:
+  description: >
+    This attribute identifies the name of the instrument or sensor that collected the data. Both
+    a long name and a short name are given. For any data file, only a single value is allowed.
+    For HERMES, the following are valid values:
+    - EEA>Electron Electrostatic Analyzer
+    - MERIT>Miniaturized Electron pRoton Telescope
+    - NEMISIS> Noise Eliminating Magnetometer In a Small Integrated System
+    - SPAN-I>Solar Probe Analyzer for Ions
+  default: null
+  required: true
+  validate: true
+  overwrite: false
+Discipline:
+  description: >
+    This attribute describes both the science discipline and sub discipline. For HERMES,
+    this value should always be "Space Physics>Magnetospheric Science."
+  default: Space Physics>Magnetospheric Science
+  required: true
+  validate: true
+  overwrite: false
+Generation_date:
```

Do we need to add this here? Just curious because we use different filename convention than its default. We need to use `source_descriptor_datatype_yyyyMMdd_vNNN`
```suggestion
File_naming_convention:
     description: Filename convention
     default: source_datatype_descriptor_yyyyMMdd
     required: false
     validate: false
     overwrite: false
 Generation_date:
```

---

Yes, in the IMAP specific global CDF attributes we do need to add file_naming_convention. That isn't an ISTP standard though, so it should go in a different file (eg `imap_global_cdf_attrs_schema.yaml`)